packages/mds-transaction-service/tests/transactions.spec.ts

```diff
@@ -123,6 +134,9 @@ describe('Transaction Service Tests', () => {
 
         it('Create one transaction with missing fee_type rejects', async () => {
           const [transaction] = transactionsGenerator(1)
+          if (!transaction) {
+            throw new Error('No sample transaction found') // unexpected to ever happen
+          }
```

Comment:
Seems like an opportunity for a function that asserts something is defined or throws that could replace all of the `if (!something) { throw new Error() }` blocks

Reply:
Mmm yes good call, I'll file a followup ticket